### PR TITLE
Chapter1 Homework1And2

### PR DESCRIPTION
**Pull request by Mahdi0123Copen**

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,15 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name = "Mahdi";
+            int age = 29;
+            double weight = 80.9;
+            float height = 166.6f;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'height'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5139827)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5139827&bid=16885681)

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,15 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name = "Mahdi";
+            int age = 29;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'age'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5139827)

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -6,7 +6,15 @@ class Program
...
+            double weight = 80.9;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'weight'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5139827)

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -6,7 +6,15 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name = "Mahdi";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'name'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5139827)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5139827&bid=16885681)

File: Src/BootCamp.Chapter/Program.cs [hunk 5]
```diff
@@ -6,7 +6,15 @@ class Program
...
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            const string name = "Mahdi";
+            const int age = 29;
+            const double weight = 80.9;
+            const float height = 166.6f;
+            Console.WriteLine($"{name} is {age} years old, his weight is {weight} kg and his height is {height} cm. ");
+            //BMI calculations
+            double bmi = weight * 703 / height / height;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'bmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5139827)

**Comment by Mahdi0123Copen** [comment]
Thanks for the feedback. I have added Console.ReadLine so it can be read from the console.

**Comment by Mahdi0123Copen** [comment]
Changed the PR name so its more descriptive

File: Src/BootCamp.Chapter/Program.cs [hunk 6]
```diff
@@ -6,7 +6,24 @@ class Program
...
+            Console.WriteLine("Please state your weight:");
+            float weight = float.Parse(Console.ReadLine());
+
+            Console.WriteLine("Please state your height:");
+            float height = float.Parse(Console.ReadLine());
+
+            Console.WriteLine($"{name} is {age} years old, his weight is {weight} kg and his height is {height} cm. ");
+
+            //BMI calculations
+            double bmi = weight * 703 / height / height;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Where does 703 come from? Is it a leftover from what you're supposed to read from a console?

Comment (Mahdi0123Copen) [reply Src/BootCamp.Chapter/Program.cs]: i found somewhere online that said thats the formula for calculating BMI.

Comment (Almantask) [reply Src/BootCamp.Chapter/Program.cs]: Don't trust random online sources and verify from multiple sources :)

**Pull request closed without merging**